mne_qt_browser/_pg_figure.py

```diff
@@ -3137,13 +3158,15 @@ def _hidpi_mkPen(*args, **kwargs):
             'home': {
                 'alias': dur_keys[0],
                 'qt_key': Qt.Key_Home,
+                'kw': 'step',
```

Comment:
This annoying thing is because the `.trigger()` in the fake mouse click function seems to cause the callback to be called with the signature `triggered(isChecked)`